helm/promtail/values.yaml

```diff
@@ -3,11 +3,21 @@ kyvernoPolicyExceptions:
   enabled: true
   namespace: giantswarm
 
+ciliumNetworkPolicy:
+  enabled: true
```

Comment:
Should this not be false by default?

Reply:
I prefer to have it to `true` by default as at some point all clusters will be using cilium when running on a v19 release. And in any case, thx to what we did with the observability-bundle it will be set according to the cluster's release